Utilities/KWStyle/ITKOverwrite.txt

```diff
@@ -1,3 +1,2 @@
-test/.*\.cxx Namespace Disable
-Examples/.*\.cxx Namespace Disable
+.*\.cxx Namespace Disable
```

Comment:
This disabling might be overly broad.

Reply:
Yes, but I think that applies to all of ITK.

Reply:
I think these namespace rules should only be enforced in the public API (Headers).